files/common/usr/bin/unpack-image

```diff
@@ -24,7 +24,7 @@ function die() {
 
 function usage() {
 	echo "$(basename "$0"): $*" >&2
-	echo "Usage: $(basename "$0") [-f] [-s] [-x] <image>"
+	echo "Usage: $(basename "$0") [-f] [-s] [-x] <image> <tmp_dir>"
```

Comment:
@vimalraj-anbarasu We'd have to add a similar script in 5.3 to have this work for migration. Note that the format for 5.3 images is slightly different (the files to extract are under the `depot` directory)

Reply:
FWIW, I'm fine if the app-gate wants to use the original two-stage approach for migration.. but that means we'd have to make the originally proposed changes for migration in 5.3, and different changes to work for "L" upgrades in master.

Reply:
Yes, that's correct. I think the changes would be very similar though, right?

Reply:
> I think the changes would be very similar though, right?

That'd be a question for @vimalraj-anbarasu.. I haven't looked closely at the app-gate side of this; but it might be. Either way, I don't think we need to try to keep 5.3 and master similar, in this respect.

One thing to keep in mind when making changes to the 5.3 app-gate upgrade code, is whether that logic will need to handle both, 5.3.x -> 5.3.y upgrades and 5.3.[x,y] -> 6.x.x migrations.. but I'd recommend we move this discussion over to slack or an app-gate review, since this isn't related to this specific PR.

Reply:
Yeah, I will try to stick to a similar solution for backport. Once I get a review out, we can discuss on it.